### DISCUSSION TITLE
How to rename the library folder name ?

### DISCUSSION THREAD

**Discussion by d4Rkzo** [jellyfin/jellyfin#8391] [Q&A]
![image](https://user-images.githubusercontent.com/98599873/189576136-3eb1addb-c8c0-4551-ab8d-2ed4600d9a51.png)
![image](https://user-images.githubusercontent.com/98599873/189576194-efa72830-ff7b-48b2-9772-5c6a236eaca3.png)
Need help to change the library name ....

**Comment by crobibero** [comment]
Server settings -> Libraries -> Click the 3 dots on the library card you want to edit.

**Reply by d4Rkzo** [reply]
I know how to change library name . What I want to know is how to change the folder name which appears after enabling "Display a folder view to show plain media folders" and contain all my library inside.
![image](https://user-images.githubusercontent.com/98599873/189982766-12060e12-71b8-4f61-9375-999e9df405b8.png)
